Sources/App/Core/RSS.swift

```diff
@@ -102,14 +94,33 @@ extension RecentRelease {
                         )
                     )
                 ),
-                .p(.text(packageSummary ?? "")),
+                body,
                 .small(
                     .a(
                         .href(packageUrl),
                         .text("\(repositoryOwner)/\(repositoryName)")
                     )
                 )
             )
+        }
+
+        return .item(
+            .guid(.text(packageUrl), .isPermaLink(true)),
+            .title("\(packageName) - \(version)"),
+            .link(packageUrl),
+            .pubDate(releasedAt, timeZone: .utc),
+            .description(
+                layout(
+                    .p(.text(packageSummary ?? ""))
+                )
+            ),
+            .content(
+                .unwrap(releaseNotesHTML) { notes in
+                    layout(
+                        .div(.raw(notes))
+                    )
+                }
+            )
```

Comment:
I noticed that all our snapshots come with 

```
<content:encoded><![CDATA[]]></content:encoded>
```

I guess because we don't have html release notes set - which we should do for at least one of them.

How does an empty `content:encoded` manifest itself in RSS readers? Will this just be ignored?

Or should we move the `unwrap` up one level?

```swift
             .unwrap(releaseNotesHTML) { notes in
                 .content(
                     layout(
                         .div(.raw(notes))
                     )
                 }
             )
```

such that the whole block is omitted if `releaseNotesHTML` is `nil`?

Reply:
I've gone ahead and made that change. Let me know that makes sense and I'll merge!

Reply:
> How does an empty content:encoded manifest itself in RSS readers?

They ignore it and use description instead, but lets be safe

> I've gone ahead and made that change. Let me know that makes sense and I'll merge!

Many thanks, that's what I've been looking for! The top level `unwrap` 👌